CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.1.0.73
```

Comment:
Will edit this on release. I think probably will remove the build number and just keep the major/minor/patch, otherwise editing the file will trigger a build itself.